libs-scala/scala-utils/src/main/scala/com/daml/scala/util/Statement.scala

```diff
@@ -1,9 +1,16 @@
 // Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.daml.http
+package com.daml.scala.util
```

Comment:
I suggest calling this `scalautil`. That's because if this is in classpath and you import `com.daml._`, you have a different `scala` in scope than the one you expect.

I hate to bikeshed, but we made this mistake elsewhere and it turned out to be more annoying than I expected, so we should probably avoid it here.